### ISSUE TITLE
[Feature] Formulaire pour les validateurs

### ISSUE THREAD

**Issue by Tarrke** [pluginsGLPI/formcreator#815]
Bonjour,

voici mon soucis, j'ai un formulaire qui doit être dématérialisé et je comptais utiliser ce plugin de GLPI pour le faire. Le formulaire papier en question contient des cases à remplir par le demandeur et des cases à remplir par le validateur. Serait-il possible de pouvoir modifier le formulaire du validateur (aujourd'hui une simple zone de texte) de manière à ce qu'il puisse renseigner ces cases (par exemple : "quelle ligne budgétaire sera impactée ?").

Merci d'avance.

**Comment by btry** [comment]
Bonjour

C'est quelque chose qui a déjà été demandé (je lierai le ticket), mais ce n'est pas implémenté pour le moment.

Si vous souhaitez avoir cette fonctionnalité rapidement, vous pouvez joindre le service commercial: contact@teclib.com .

**Comment by Tarrke** [comment]
Bonjour,

si jamais je venais à implémenter la fonctionnalité seriez vous intéressés?

**Comment by btry** [comment]
Oui, je trouve que la fonctionnalité est assez pertinente. Si vous avez compétence et disponibilité vous pouvez vous lancer. 

Comme il s'agit d'une fonctionnalité, je ne fusionnerai pas avant la version 2.7.0. Ca donne un peu de temps. Si vous avez des questions, besoin d'aide, demandez de préférence sur la liste de diffusion développeurs sur OW2 (j'y suis abonné).

Quelques directives:

## code d'upgrade 
vous aurez certainement des changements sur la structure des tables. Créez un fichier install/update_dev.php contenant une fonction plugin_formcreator_update_dev(). Dedans mettez votre code d'upgrade. Il se pourrait que je bascule cela en classes bientôt. Je tâche de vous en informer.

## tests unitaires
Si il est possible de tester votre code, ajoutez des tests unitaires.
